src/script.js

```diff
@@ -839,7 +839,7 @@ document.addEventListener('DOMContentLoaded', function () {
 });
 
 const addFloatingComposeButton = () => {
-  if (select.composeButtonOld()) {
+  if (!document.querySelector('.floating-compose')) {
```

Comment:
Can we move this up to the `select` object for consistency?

Reply:
Good call, that comment made me connect the dots, please see new commit